server/http_hooks.go

```diff
@@ -144,12 +144,17 @@ func (p *Plugin) handleDialog2(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	suffix := ""
+	if request.State == "relative-callback-url" {
```

Comment:
Would you please make this a constant? It's used in three locations.

Reply:
Sure, I added dialog state constants.  Please take a look again.